__tests__/Port.test.js

```diff
@@ -0,0 +1,38 @@
+/* globals describe it expect */
+const Port = require('../src/Port.js');
+
+describe('Port', () => {
+    describe('with ports and ships', () => {
```

Comment:
I would only nest describe blocks if you're going to use them to separate out different test setups/conditions, e.g. have 2 or more describe blocks inside of your main describe block. Otherwise, you end up just over indenting your code making it harder to fit on the screen.